src/main/resources/db/migration/V1_2__visit_id.sql

```diff
@@ -0,0 +1,9 @@
+create table visit_id
+(
+    nomis_id bigint       not null  constraint visit_id_nomis_id_pkey   primary key,
+    vsip_id  varchar(80)  not null  constraint visit_id_vsip_id_unique  unique,
```

Comment:
is this just a guess or are their IDs going to be 80 characters long?

Reply:
just a guess, can probably make it shorter